package.json

```diff
@@ -36,28 +36,47 @@
     "react-router-hash-link": "^2.4.3",
     "react-to-print": "^2.15.1",
     "react-visibility-sensor": "^5.1.1",
+    "swiper": "^11.1.4",
```

Comment:
It looks like this was re-added, when it has been removed from the latest develop.